<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFullScreenSourceDlg</class>
 <widget class="QDialog" name="AddFullScreenSourceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddFullScreenSourceDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#262626;
border:1px solid #585858;
}
#wgt_title{
background:#2F2F2F;
}
QLabel,QPushButton{
background:transparent;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
}
QComboBox{
background:transparent;
border:1px solid #585858;
border-radius:4px;
padding-left:9px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
}
QComboBox::disabled{
border:1px solid #2F2F2F;
color:#777777;
}
 QComboBox:on { 
      padding-top: 3px;
      padding-left: 4px;
  }
QComboBox::drop-down{
subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 30px;
      border:none;
}
QComboBox::down-arrow{
image: url(:/TQLive/images/live tool_icon_down.png);
}
QComboBox::down-arrow:on {
      top: 1px;
      left: 1px;
  }
QListView{
background:#262626;
border:1px solid #585858;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
outline:0px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
padding-bottom:2px;
color:#999999;
}
QListView::item{
background:#262626;
color:#999999;
min-height:30px;
}
QListView::item:hover{
background:#2A2A2A;
color:#4AB134;
}
#button_confirm{
background:transparent;
background-image: url(:/TQLive/images/btn_confirm.png);
background-repeat:no-repeat;
border:none;
}
#button_confirm:hover{
background-image: url(:/TQLive/images/btn_confirm_hover.png);
}
#button_confirm:pressed{
margin:1px;
padding:1px;
}
#button_cancel{
background:transparent;
background-image: url(:/TQLive/images/btn_cancel.png);
background-repeat:no-repeat;
border:none;
}
#button_cancel:hover{
background-image: url(:/TQLive/images/btn_cancel_hover.png);
}
#button_cancel:pressed{
margin:1px;
padding:1px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QWidget" name="wgt_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_title{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#4AB134;
background:transparent;
}
#button_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#button_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#button_close:pressed{
margin:1px;
padding:1px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>19</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>全屏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_close">
        <property name="minimumSize">
         <size>
          <width>12</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>39</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_selected">
         <property name="text">
          <string>请选择显示器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_selected">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_tips">
         <property name="text">
          <string>全屏直播对CPU消耗较大，若无明确需求，建议使用其他直播方式</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>19</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_confirm">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="minimumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>82</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
